test/Feature/IgnoreAclOpenApiSpec.hs

```diff
@@ -1,65 +0,0 @@
-module Feature.IgnorePrivOpenApiSpec where
```

Comment:
Renamed this file on https://github.com/PostgREST/postgrest/pull/1882. Forgot to delete it :man_facepalming:.